src/index.js

```diff
@@ -1,23 +1,14 @@
-(() => {
+
+async function main() {
```

Comment:
do we need this wrapper anymore? iirc with esm we don't anymore

Reply:
removed wrapper